packages/page-accounts/src/modals/Create.tsx

```diff
@@ -240,9 +252,9 @@ function Create ({ className = '', onClose, onStatusChange, seed: propsSeed, typ
 
   const _onChangePairType = useCallback(
     (newPairType: PairType) => setAddress(
-      updateAddress(seed, '', seedType, newPairType)
+      updateAddress(seed, isEthereum ? derivePath : '', seedType, newPairType)
```

Comment:
```suggestion
      updateAddress(seed, isEthereum ? ETH_DEFAULT_PATH : '', seedType, newPairType)
```

Since we are swapping from something else to Ethereum, the path cannot be the previous path, but needs to be ETH-specific.